parcels/kernels/diffusion.py

```diff
@@ -0,0 +1,25 @@
+from parcels.rng import random
+import math
+
+
+__all__ = ['SaptiallyVaryingDiffusion2D']
+
+
+def SaptiallyVaryingDiffusion2D(particle, grid, time, dt):
+    # Diffusion equations for particles in non-uniform diffusivity fields
+    # from Ross &  Sharples 2004 and Spagnol et al. 2002
+    to_lat = 1 / 1000. / 1.852 / 60.
+    to_lon = to_lat / math.cos(particle.lat*math.pi/180)
```

Comment:
I am very worried about these kind of explicit geographic statements in default kernels! This will not work for users who run with flat meshes. It's probably acceptable to have these explicit `to_lon` statements in custom kernels in example scripts, but the kernels in `parcels/kernels/*` should be generic w.r.t. meshes

So we need to get this to work through the unit-converter. I'll have a look in the coming days